### ISSUE TITLE
Audio playback not working for `no-buttons` select choices

### ISSUE THREAD

**Issue by seadowg** [getodk/collect#3354]
#### Software and hardware versions 
Collect v1.23.3

#### Problem description
When clicking on a select choice for a prompt that's using the `no-buttons` appearance (previously `compact`) there is an error if the select choice has attached audio.

#### Steps to reproduce the problem
1. Load [this example form](https://github.com/opendatakit/collect/files/3626538/gridwidget.zip)
2. Click on one of the options in the first question

#### Expected behavior
Previously the app would have played the audio when the choice was made - but only if `no-buttons` or `compact` was used. The easiest fix right now would be to simply remove the old code that's attempting to play the audio in this one case. Alternatively we can implement the new audio playback in `GridWidget` as we've already done for `SelectWidget` and `QuestionWidget`.

**Comment by grzesiek2010** [comment]
It's not very important I think we should prioritize https://github.com/opendatakit/collect/issues/3300 and make decisions about that issue first.

**Comment by seadowg** [comment]
@grzesiek2010 that makes sense. It might not be worth fixing if we get rid of the `GridWidget`. 

My worry would be that the new audio implementation will go out before we can make changes there and so we'd end up breaking any forms that use audio with the `GridWidget`.

**Comment by lognaturel** [comment]
Did it work previously?

**Comment by seadowg** [comment]
Yeah looks like it's working in v1.23.3 - just checked using the Play Store version I have on my own phone. Given we're getting close to a release it seems like it's a good idea to fix this to prevent any forms that are using this feature from breaking. We can also include an analytics event for this case as it would be nice to know how many folks are using it.

**Comment by seadowg** [comment]
@opendatakit-bot claim
